tests/sources/xml/test_ead.py

```diff
@@ -9,29 +8,27 @@
 
 
 def create_ead_source_record_stub(
-    header_insert: str = "", metadata_insert: str = ""
+    header_insert: str = "",
+    metadata_insert: str = "",
+    parent_element: Literal["archdesc", "did"] = None,  # noqa: RUF013
```

Comment:
This isn't too complex in my mind and saves space in the unit tests but I defer to @ghukill for final approval of this